lib/spack/spack/cmd/style.py

```diff
@@ -121,10 +122,6 @@ def changed_files(base="develop", untracked=True, all_files=False, root=None):
         files = git(*arg_list, output=str).split("\n")
 
         for f in files:
-            # Ignore non-Python files
-            if not (f.endswith(".py") or f == "bin/spack"):
```

Comment:
Technically `bin/spack` is both POSIX shell and Python. Since the script is tiny and contains multiple languages, I'm not particularly concerned about missing it during style testing.